crates/primitives/commitments/Cargo.toml

```diff
@@ -43,4 +43,4 @@ std = [
 ]
 parity-scale-codec = ["dep:parity-scale-codec", "mp-felt/parity-scale-codec"]
 scale-info = ["dep:scale-info", "mp-felt/scale-info"]
-serde = ["dep:serde", "dep:starknet-ff", "starknet-ff/serde", "mp-felt/serde"]
```

Comment:
I don't think this is good.

serde = ["dep:serde", "dep:starknet-ff", "starknet-ff?/serde", "mp-felt/serde"]

Reply:
But we are going to nuke this anyway. Replace it by our bonzai impl